src/OVAL/probes/unix/linux/rpm-helper.c

```diff
@@ -19,11 +19,100 @@
  */
 
 #include "rpm-helper.h"
-
+#include <errno.h>
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
+
+
+void probe_chroot_init(struct probe_chroot* ch, const char* scan_path) {
```

Comment:
Please follow our coding style and be consistent with the coding style. Thank you.